build/build.proj

```diff
@@ -12,6 +13,7 @@
   <Target Name="CI">
     <PropertyGroup>
       <Configuration>Release</Configuration>
+      <Platform>&quot;Any CPU&quot;</Platform>
```

Comment:
:bulb: This line isn't necessary, because only one platform is ever used by the build and the definition added above will be applied in all cases. I believe duplicating the definition increases the risk that the property will become incorrect in the future.